python/cudf/cudf/core/dataframe.py

```diff
@@ -6370,6 +6370,8 @@ def to_parquet(
         max_page_size_rows=None,
         storage_options=None,
         return_metadata=False,
+        use_dictionary=True,
```

Comment:
Can we document these two parameters here: https://github.com/rapidsai/cudf/blob/16051a718509c218010e6912d6f8e0fca7a7aa24/python/cudf/cudf/utils/ioutils.py#L222

Reply:
done

Reply:
Thanks @galipremsagar. I would have never thought to look there for the docstring :sweat_smile:

Reply:
First time hearing about it as well :man_shrugging: (just don't git blame `ioutils.py` :P )